docs/getting-started/document-conventions.md

```diff
@@ -0,0 +1,60 @@
+---
+sidebar_position: 4
+sidebar_label: Document Conventions
+title: "Document Conventions"
+keywords:
+- Harvester
+- documentation
+- conventions
+- labels
+---
+
+<head>
+  <link rel="canonical" href="https://docs.harvesterhci.io/v1.4/getting-started/document-conventions"/>
+</head>
+
+## Release Labels
+
+Harvester has a 26-week release cycle and an N - 1 support policy. Each release is supported for 14 months (12 months of support and 2 months for upgrades). Only the two most recent minor versions receive security and bug fixes. For more information, see the [Support Matrix](https://www.suse.com/suse-harvester/support-matrix/all-supported-versions/).
```

Comment:
@innobead I think we should change this for 1.5.0 accordingly. The release cycle is tuned to a 4-month pace. And maybe also mention the community versions vs prime versions?

Reply:
Yes, we should.